delete_user_V2

```diff
@@ -1,37 +1,55 @@
-
 <!DOCTYPE html>
 <html lang = "en">
+	
 	<head>
+		
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+		
 		<meta charset = "utf-8" />
 		<title>Delete User</title>
+		
+		<!-- Link to CSS files for styling the page -->
 		<link rel="stylesheet" type="text/css" href="css/queries.css">
 		<link rel="stylesheet" type="text/css" href="css/style.css">
+		
 	</head>
+	
```

Comment:
When moving on to the second version of my delete user page, I aimed to improve the layout. This time I made the input type sections where the user could input their username and password to be placeholders. Then I added some style to this placeholder to make it look nicer. I created a style section at the top of the page before I started the body tag. Here I gave the text color white and padding for spacing.